introduction.md

```diff
@@ -5,3 +5,5 @@ My favourite codeschool part is SQL, Mongo DB, Python, HTML & CSS.
 *Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?